src/jit/function-builder.cpp

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2017 wasmjit-omr project participants
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "function-builder.hpp"
+#include "src/interp.h"
+#include "ilgen/VirtualMachineState.hpp"
+#include <type_traits>
+
+wabt::jit::FunctionBuilder::FunctionBuilder(interp::Thread* thread, interp::IstreamOffset const offset, TypeDictionary* types)
+    : TR::MethodBuilder(types)
+    , thread_{thread}
+    , offset_{offset}
+    , valueType_{types->LookupUnion("Value")}
+    , pValueType_{types->PointerTo(types->LookupUnion("Value"))}
+{
+    DefineLine(__LINE__);
+    DefineFile(__FILE__);
+    DefineName("WASM_Function");
+
+
+    DefineReturnType(types->toIlType<std::underlying_type<wabt::interp::Result>::type>());
+//    setUseBytecodeBuilders();
+//    AllLocalsHaveBeenDefined();
```

Comment:
I'm not sure what to make of these commented out calls. If this was intentional, please leave a TODO documenting why they've been commented out to make sure they get re-added when necessary.

Reply:
Those are calls I originally added because we will (probably) eventually need them, just not at the moment.

I've removed them for now as they really should be added as part of the work that needs them.